documentation/schema.json

```diff
@@ -1335,6 +1345,7 @@
       "enum": [
         "CollectDump",
         "CollectGCDump",
+        "CollectLiveMetrics",
```

Comment:
I know that the route name is `livemetrics`, but I'm wondering if the action should just be called `CollectMetrics`. I would think that there won't be any other type of metrics collection action that would need to be differentiated from this one.

Reply:
Even if we aren't planning an action for metrics, I think that might introduce some confusion in terms of the expected type being returned (e.g.  we say "Metrics is Prometheus, and LiveMetrics is Json, _except_ in the case of the collection rules, where metrics is Json"). That being said, if people want me to change it, I can - just my 2 cents.